config/connect-log4j.properties

```diff
@@ -13,13 +13,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-log4j.rootLogger=INFO, stdout
+log4j.rootLogger=INFO, stdout, ROLLINGFILE
 
 
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
+#
+# Add ROLLINGFILE to rootLogger to get log file output
+#    Log DEBUG level and above messages to a log file
+log4j.appender.ROLLINGFILE=org.apache.log4j.ConsoleAppender
+log4j.appender.ROLLINGFILE.Threshold=DEBUG
+log4j.appender.ROLLINGFILE.File=bookkeeper.log
```

Comment:
This also doesn't follow currently-established conventions. For example, the `log4j.properties` file uses:

    log4j.appender.kafkaAppender.File=${kafka.logs.dir}/server.log

and if we follow that convention then the line in this file might be:

    log4j.appender.connectAppender.File=${kafka.logs.dir}/connect.log

The `kafka-run-class.sh` script sets the `${kafka.logs.dir}` system property [here](https://github.com/apache/kafka/blob/trunk/bin/kafka-run-class.sh#L154-L175).